connect/cli/plugins/product/sync/items.py

```diff
@@ -274,6 +282,12 @@ def _get_or_create_unit(self, data):
             if unit['type'] == data.type and unit['description'] == data.unit:
                 return unit['id']
 
+        for unit in self._shared_units:
+            if unit['id'] == data.unit:
+                return unit['id']
+            if unit['type'] == data.type and unit['description'] == data.unit:
```

Comment:
In export name of unit is written, so it is more clear if data will be compared by name, not description. WDYT?

Reply:
the unit collection return the following representation:

```
    {
        "id": "CU-876-577",
        "unit": "unit",
        "description": "Boolean",
        "owner": {
            "id": "VA-037-000",
            "name": "Vendor account 00 for ffaraone"
        },
        "type": "reservation",
        "cumulative": false
    },
    {
        "id": "boolean",
        "unit": "unit",
        "description": "Boolean",
        "type": "reservation",
        "cumulative": false
    }
```

There is no name attribute there. But the unit representation inside the product item representation call it `name`.

Reply:
Oh, I understood